subsys/shell/shell_ops.c

```diff
@@ -369,7 +369,13 @@ static void shell_pend_on_txdone(const struct shell *shell)
 {
 	if (IS_ENABLED(CONFIG_MULTITHREADING) &&
 	    (shell->ctx->state < SHELL_STATE_PANIC_MODE_ACTIVE)) {
-		k_poll(&shell->ctx->events[SHELL_SIGNAL_TXDONE], 1, K_FOREVER);
+		struct k_poll_event event;
```

Comment:
It looks like `shell->ctx->events[SHELL_SIGNAL_TXDONE]` is now unused with this change.   
Can you add a comment to the `struct shell_context` definition to indicate that the TXDONE event is not used?

Reply:
Sure, added.

Reply:
@semihalf-barnas-michal 
I think that Keith was suggesting to leave a comment here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/include/shell/shell.h#L701

Reply:
@jakub-uC : That's where I have written comment. I can see it here, so I think that PR was updated correctly:
https://github.com/zephyrproject-rtos/zephyr/pull/38705/files

Reply:
I'm good with the comment added by Michal.